recipe/meta.yaml

```diff
@@ -124,7 +127,6 @@ outputs:
     version: {{ version }}
     build:
       noarch: python
-      number: {{ build }}
```

Comment:
Any reason not to add these explicitly in each output? Is it just a simplification or are we trying to prevent some faulty behavior?

Reply:
just a simplification. I think the bot also has trouble with either "build" or "number" and I never remember which one it fails to bump automatically.